torch_xla/csrc/data_ops.h

```diff
@@ -5,6 +5,7 @@
 #include "tensorflow/compiler/xla/client/xla_builder.h"
 #include "tensorflow/core/lib/gtl/array_slice.h"
 #include "torch/csrc/jit/ir.h"
+#include "torch_xla/csrc/translator.h"
```

Comment:
Translator depends on data_ops, and data_ops depends on translator?

Reply:
Translator header doesn't depend on data_ops, only its implementation.

Reply:
OK for this, but the core translator data structures should be in a separate file for cleaner dependency graph.

Reply:
Better? We only need the `SizeOpValues` alias, seems excessive to create a new header for it. Maybe we could move `XlaComputationInOut` and `XlaTranslationResult` to a separate header, but I don't think there's a good justification for it besides the availability of `SizeOpValues` alias.